Submodule platform contains modified content
diff --git a/platform/toolkit/mozapps/installer/packager-uxp.mk b/platform/toolkit/mozapps/installer/packager-uxp.mk
index 8d7cb4e62..7b5dda4a4 100644
--- a/platform/toolkit/mozapps/installer/packager-uxp.mk
+++ b/platform/toolkit/mozapps/installer/packager-uxp.mk
@@ -18,6 +18,6 @@ else
 ifeq (WINNT,$(OS_ARCH))
 	cd $(DIST); $(CYGWIN_WRAPPER) 7z a -t7z -m0=lzma2 -mx=9 -aoa -bb3 $(PKG_BASENAME).7z $(MOZ_PKG_DIR)
 else
-	cd $(DIST); XZ_OPT=-9e $(TAR) cfJv $(PKG_BASENAME).tar.xz $(MOZ_PKG_DIR)
+	cd $(DIST); $(TAR) cfv $(PKG_BASENAME).tar $(MOZ_PKG_DIR)
 endif
 endif
diff --git a/platform/toolkit/mozapps/installer/packager.mk b/platform/toolkit/mozapps/installer/packager.mk
index 6cae15658..760586367 100644
--- a/platform/toolkit/mozapps/installer/packager.mk
+++ b/platform/toolkit/mozapps/installer/packager.mk
@@ -230,7 +230,7 @@ source-package:
 	# Don't bother removing it as this is only used by automation.
 	@cp $(MOZ_SOURCESTAMP_FILE) '$(MOZ_PKG_SRCDIR)/sourcestamp.txt'
 	$(MKDIR) -p $(DIST)/$(PKG_SRCPACK_PATH)
-	(cd $(MOZ_PKG_SRCDIR) && $(CREATE_SOURCE_TAR) - ./ ) | xz -9e > $(SOURCE_TAR)
+	(cd $(MOZ_PKG_SRCDIR) && $(CREATE_SOURCE_TAR) - ./ ) > $(SOURCE_TAR)
 
 hg-bundle:
 	$(MKDIR) -p $(DIST)/$(PKG_SRCPACK_PATH)
diff --git a/platform/toolkit/mozapps/installer/upload-files.mk b/platform/toolkit/mozapps/installer/upload-files.mk
index 1bbccecb2..4dd04b369 100644
--- a/platform/toolkit/mozapps/installer/upload-files.mk
+++ b/platform/toolkit/mozapps/installer/upload-files.mk
@@ -490,7 +490,7 @@ endif
 CREATE_SOURCE_TAR = $(TAR) -c --owner=0 --group=0 --numeric-owner \
   --mode=go-w $(SRC_TAR_EXCLUDE_PATHS) --transform='s,^\./,$(SRC_TAR_PREFIX)/,' -f
 
-SOURCE_TAR = $(DIST)/$(PKG_SRCPACK_PATH)$(PKG_SRCPACK_BASENAME).tar.xz
+SOURCE_TAR = $(DIST)/$(PKG_SRCPACK_PATH)$(PKG_SRCPACK_BASENAME).tar
 HG_BUNDLE_FILE = $(DIST)/$(PKG_SRCPACK_PATH)$(PKG_BUNDLE_BASENAME).bundle
 SOURCE_CHECKSUM_FILE = $(DIST)/$(PKG_SRCPACK_PATH)$(PKG_SRCPACK_BASENAME).checksums
 SOURCE_UPLOAD_FILES = $(SOURCE_TAR)
